src/dymaptic.GeoBlazor.Core/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "dymaptic.GeoBlazor.Core",
-  "version": "2.3.0",
+  "version": "2.3.1-beta-1",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "dymaptic.GeoBlazor.Core",
-      "version": "2.3.0",
+      "version": "2.3.1-beta-1",
```

Comment:
what is the difference between 2.3.1-beta-1 and everywhere else that has 2.3.2?

Reply:
package-lock.json should generate on build from package.json, guess I forgot to do a build here, will correct before merging!